flag.go

```diff
@@ -778,7 +778,7 @@ func visibleFlags(fl []Flag) []Flag {
 func prefixFor(name string) (prefix string) {
 	if len(name) == 1 {
 		prefix = "-"
-	} else {
+	} else if name[0] != '-' {
```

Comment:
Did you mean to include this change? It seems unrelated.

Reply:
Oops, yeah, I don't even remember why I did that, but I needed it for something, removed it tho.